### PR TITLE
ATM2018Q2 - Hometask M8

### PR DESCRIPTION
**Pull request by yauhen07**
Draft

### CODE REVIEW

File: src/main/java/businessobject/User.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package businessobject;
```
Comment (evgeny-tumashchick) [inline src/main/java/businessobject/User.java]: ..objectS

File: src/main/java/util/DataGeneration.java [hunk 2]
```diff
@@ -4,13 +4,11 @@
 
     public static String generateUniqueTradeName() {
         int x = (int) (Math.random() * 1000);
-        String tradeName = "yauhen21" + x;
-        return tradeName;
+        return "yauhen21" + x;
```
Comment (evgeny-tumashchick) [inline src/main/java/util/DataGeneration.java]: Recommended : Move yauhen21 to constant. And in next method

**Review by evgeny-tumashchick — APPROVED** [review]

File: src/main/java/businessobject/User.java (hunk 1 above)
Comment (yauhen07) [reply src/main/java/businessobject/User.java]: done

File: src/main/java/util/DataGeneration.java (hunk 2 above)
Comment (yauhen07) [reply src/main/java/util/DataGeneration.java]: Done

**Pull request merged**